diff --git a/src/panfrost/perf/quick.c b/src/panfrost/perf/quick.c
index a91b92f85c8..c157ed53333 100644
--- a/src/panfrost/perf/quick.c
+++ b/src/panfrost/perf/quick.c
@@ -43,7 +43,7 @@ main(void)
         ctr++) {
       int64_t val = pan_perf_counter_read_sum(perf, ctr);

-      printf("%s: %ld\n", ctr->name, val);
+      printf("%s: %ld\n", ctr->name, (long)val);
    }

    if (pan_perf_disable(perf) < 0) {
